PaintingTrivia.py

```diff
@@ -116,6 +120,21 @@ def main_menu():
     return render_template('main_menu.html', username=session['username'])
 
 
+@app.route('/results', methods=['GET', 'POST'])
+def show_quiz_results():
+    if request.method == 'POST':
+        # Clear session
+        session.pop('quiz')
+        x = layout_buttons()
+        if x:
+            return x
+    quiz = pickle.loads(session['quiz'])
```

Comment:
i don't like it. i think we need to install yet another library http://flask.pocoo.org/docs/0.10/api/#sessions